config/routes.rb

```diff
@@ -300,6 +300,10 @@
     scope '/verify', as: 'idv' do
       get '/' => 'idv#index'
       get '/activated' => 'idv#activated'
+      if !FeatureManagement.idv_available?
+        # IdV has been disabled.
+        match '/*path' => 'idv#unavailable', via: %i[get post]
+      end
```

Comment:
Since we’re moving IdV out into many controllers (moving away from FSM), I went with a blanket check at the router level rather than adding action filters to individual controllers or adding a new Idv “Base” controller.

Reply:
Smooth!